packages/relic/lib/src/headers.dart

```diff
@@ -0,0 +1,1219 @@
+import 'dart:collection';
```

Comment:
@hampuslavin 
In the `headers` folder, I only include custom headers and the authorization header. The idea is to keep everything custom that's used specifically within this file, so this it the main headers file.
 Do you think it would be a good idea to include this as well?

Reply:
No I think it's fine to keep as is 👍